classes/issue/Issue.inc.php

```diff
@@ -444,7 +444,7 @@ function setCoverImageAltText($coverImageAltText, $locale) {
 	 * @param $force array force show/hide of data components
 	 * @return string
 	 */
-	function getIssueIdentification($force = array()) {
+	function getIssueIdentification($force = array(), $locale = null) {
```

Comment:
Could you add this new parameter to the self-documentation in the function header?

Reply:
Done